src/components/social-cards.js

```diff
@@ -1,34 +1,44 @@
 import React from 'react';
 import Helmet from 'react-helmet';
 
-const SocialCards = ({isBanner, title, summary}) => (
+const SocialCards = ({ isBanner, title, summary }, key) =>
```

Comment:
Shouldn't `key` be part of the prop object here? 🤔 

```suggestion
const SocialCards = ({ isBanner, title, summary, key }) =>
```

Reply:
It could be, its optional